package.json

```diff
@@ -72,13 +73,14 @@
     "crypto-browserify": "^3.12.0",
     "google-protobuf": "^3.12.2",
     "grpc-web": "1.2.1",
+    "isomorphic-ws": "^4.0.1",
     "os-browserify": "^0.3.0",
     "stream-browserify": "^3.0.0",
     "url": "^0.11.0",
     "util": "^0.12.3",
-    "ws": "^7.3.1",
+    "ws": "^7.4.0",
     "xhr2": "0.2.0",
-    "xpring-common-js": "6.2.7"
+    "xpring-common-js": "xpring-eng/xpring-common-js#cbdc-demo"
```

Comment:
This should be updated once we release the changes in `xpring-common-js`.

Reply:
gotcha, which is why you suggested first cutting a release of xpring-common-js

Reply:
yup